src/ocean/util/config/ConfigFiller.d

```diff
@@ -1047,7 +1038,7 @@ private void readFieldsImpl ( T, Source )
 
         if ( config.exists(group, key) )
         {
-            static if ( is(Type U : U[]) && !isStringType!(Type))
+            static if (is(Type U : U[]) && !isUTF8StringType!(Type))
```

Comment:
I'm surprised you keep using `is(Type U : U[])` here.

Reply:
My primary goal was right now is to stop using `core.Traits`, not improve generic facilities in general (that is a bit more complicated task), so I haven't been paying much attention to cases like this.

Reply:
Fair enough.